spinalcordtoolbox/scripts/sct_label_utils.py

```diff
@@ -104,8 +104,8 @@ def get_parser():
     func_group.add_argument(
         '-create-viewer',
         metavar=Metavar.list,
-        type=list_type(',', int),
-        help="Manually label from a GUI a list of labels IDs, separated with ','. Example: 2,3,4,5"
+        help="Manually label from a GUI a list of labels IDs, either created from an interval "
+             "(Example 2:5 which is equivalent to 2,3,4,5) or separated with ',' (Example: 2,3,4,5)"
```

Comment:
With the current wording, it might not be clear to users that they can also provide a mix of intervals and individual values.

So maybe the wording could be:

```suggestion
        help="Manually label from a GUI a list of labels IDs. Provide a comma-separated list"
             "containing individual values and/or intervals. Example: '-create-viewer 1:4,6,8'
             "will allow you to add labels [1,2,3,4,6,8] using the GUI."
```

Reply:
Yes good idea !

Reply:
Ah, I am so sorry, my "Suggested change" had a typo, it was missing a `"`. :sweat_smile: